conf/cqlshrc.sample

```diff
@@ -33,6 +32,10 @@
 ; classname = PlainTextAuthProvider
 ; username = user1
 
+[protocol]
+;; Specify a specific protcol version otherwise the client will default and downgrade as necessary
```

Comment:
@neshkeev is this really relevant? can you point me to the PR suggesting to add this?

Reply:
@smiklosovic , [here](https://github.com/apache/cassandra/pull/366/files#diff-f55a268d347d301121adb03b1d33958837ed984295b83fbe474dde2781fe809fR26) it is